php/libraries/NDB_Config.class.inc

```diff
@@ -447,7 +447,13 @@ class NDB_Config
         // nothing in the config file, so get the value from the DB
         // This will throw a ConfigurationException if it does not
         // exist.
-        return $this->getSettingFromDB($name);
+        try {
+            return $this->getSettingFromDB($name);
+        } catch (ConfigurationException $e) {
```

Comment:
There might be places where that exception is expected. Returning null would be unexpected.
If not, then the comment above need to be changed

Reply:
Done, this no longer returns null

Reply:
But is still catches the exception.

Reply:
Ah okay I misunderstood. Fixed now